src/databricks/labs/ucx/account.py

```diff
@@ -88,6 +89,59 @@ def sync_workspace_info(self):
                 path = f"{installation.path}/{self.SYNC_FILE_NAME}"
                 ws.workspace.upload(path, info, overwrite=True, format=ImportFormat.AUTO)
 
+    def create_account_level_groups(self):
+        """
+        Crawl all workspaces, and create account level groups if a WS local group is not present in the account
+        """
+        acc_groups = self.get_account_groups()
+
+        all_valid_workspace_groups = self.get_valid_workspaces_groups()
+
+        for group_name, group in all_valid_workspace_groups.items():
+            if group_name in acc_groups:
+                logger.info(f"Group {group_name} already exist in the account, ignoring")
+            else:
+                self._ac.groups.create(display_name=group_name, members=group.members)
+                logger.info(f"Group {group_name} created in the account")
+
+    def get_valid_workspaces_groups(self) -> dict[str, Group]:
+        all_workspaces_groups: dict[str, Group] = {}
+        inconsistent_groups: list[str] = []
+        for client in self.workspace_clients():
+            ws_group_ids = client.groups.list(attributes="id")
+            for group_id in ws_group_ids:
+                full_workspace_group = client.groups.get(group_id.id)
+                group_name = full_workspace_group.display_name
```

Comment:
If two groups in two different workspaces have the same name, but different members, add workspace name as a prefix and create group with those members on the account